### DISCUSSION TITLE
Admin area disappear after upgrading to newest biigle

### DISCUSSION THREAD

**Discussion by cywhale** [biigle/core#531] [Q&A] [answered]
After upgrading to newest release of Biigle (and merge gpu branch), all seem to work except "Admin area" disappear.
It's a 500 error as following, but only this /admin got such error. 

![https://github.com/cywhale/ODB/blob/master/ROV/biigle/clip_2023_01.jpg](https://raw.githubusercontent.com/cywhale/ODB/master/ROV/biigle/clip_2023_01.jpg)

In docker compose logs, I can only find the error (But have no idea why and how to fix it, as well as not sure if it relates to the question itself):
```
biigle-database-1    | 2023-01-11 15:08:36.009 UTC [79] STATEMENT:  select * from "announcements" where ("show_until" is null or "show_until" > $1) limit 1
biigle-database-1    | 2023-01-11 15:08:36.011 UTC [79] ERROR:  relation "announcements" does not exist at character 15

```

Seems no similar conditions had happened in issues or google. Any tips/comments to debug it would be appreciated. Thanks.

**Comment by mzur** [comment, marked as answer]
Please run the migrations as described in the [admin documentation](https://biigle-admin-documentation.readthedocs.io/maintenance/#updating).

**Reply by cywhale** [reply]
Thanks for the suggestion. I missed the .env.example updates. I have updated it to my .env, re-build, ./artisan migrate, and then docker compose restart
Previous biigle-database-1 error does not happen though,  I still get 500 error for /admin
In docker compose logs, there is an error message seems to relate it:

```
biigle-web-1         | 2023/01/12 08:48:01 [error] 31#31: *45 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 192332312 bytes) in /var/www/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php on line 56" while reading response header from upstream, client: 172.19.0.1, server: example.com, request: "GET /admin HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "MyHost.com", referrer: "https://MyHost.com/"
```
In my  docker-compose.yaml, I tried to change php memory_limit, but not work
```
command: "php -d memory_limit=4G artisan queue:work --queue=high,default --sleep=5 --tries=3 --timeout=0"
```

**Reply by mzur** [reply]
Thanks for the hint with the `.env.example` file. I've added this to the update instructions.

As to the other error, please delete the log files in `storage/logs`. It looks like the log file got too big and can't be parsed within the memory limit.

**Reply by cywhale** [reply]
Thank you. I didn't guess that the large log files caused this problem. Ya Admin area works now!
